tests/acceptance/job/basic-layout-test.js

```diff
@@ -60,7 +59,9 @@ module('Acceptance | job/basic layout', function (hooks) {
 
     assert.equal(document.title, 'Job #1234.1 - travis-ci/travis-web - Travis CI');
 
-    assert.equal($('head link[rel=icon]').attr('href'), getFaviconUri('green'), 'expected the favicon data URI to match the one for passing');
+    // Ember-test-helpers find does not work here
+    const iconHref = window.document.querySelector('head link[rel=icon]').getAttribute('href');
+    assert.equal(iconHref, getFaviconUri('green'), 'expected the favicon data URI to match the one for passing');
```

Comment:
Same here with qunit-dom

Reply:
Got the same result here. qunit-dom was unable to find the element, and it broke the test. Maybe because it's in the `head`? Well, for now I think I'm going to leave these as-is, but thanks for the suggestion! It was at least worth trying :smile: